src/bundle/DependencyInjection/Configuration/Parser/SiteAccessAware/Autocomplete.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\Search\DependencyInjection\Configuration\Parser\SiteAccessAware;
+
+use Ibexa\Bundle\Core\DependencyInjection\Configuration\AbstractParser;
+use Ibexa\Bundle\Core\DependencyInjection\Configuration\SiteAccessAware\ContextualizerInterface;
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+
+/**
+ * Configuration parser for search.
+ *
+ * @Example configuration:
+ *
+ * ```yaml
+ * ibexa:
+ *   system:
+ *      default: # configuration per siteaccess or siteaccess group
+ *          search:
+ *              autocomplete:
+ *                  min_search_test_length: 3
+ *                  result_limit: 5
+ * ```
+ */
+class Autocomplete extends AbstractParser
```

Comment:
```suggestion
final class AutocompleteParser extends AbstractParser
```

Reply:
Mentioned code is probably older. I would still suggest to make class final and improve naming by adding `Parser` suffix.